sdk/src/crypto/cose/ocsp.rs

```diff
@@ -68,7 +73,36 @@ pub fn check_ocsp_status(
             OcspFetchPolicy::FetchAllowed => {
                 fetch_and_check_ocsp_response(sign1, data, ctp, tst_info, validation_log)
             }
-            OcspFetchPolicy::DoNotFetch => Ok(OcspResponse::default()),
+            OcspFetchPolicy::DoNotFetch => {
+                if let Some(ocsp_response_ders) = ocsp_responses {
+                    for ocsp_response_der in ocsp_response_ders {
```

Comment:
Use a temporary StatusTrack in the loop.  Only merge in the results of the last loop that way you do not pollute the validation_log in a bunch of results we don't care about.  See how logging is done in verify_time_stamp.